karma.conf.js

```diff
@@ -19,15 +27,18 @@ module.exports = function (config) {
       reports: [ 'html', 'lcovonly' ],
       fixWebpackSourcePaths: true
     },
+    coverageReporter: {
+      reporters: coverage_reporters,
+    },
     angularCli: {
       environment: 'dev'
     },
-    reporters: ['progress', 'kjhtml'],
+    reporters: ['progress', 'kjhtml', 'coverage-istanbul', 'coverage', 'coveralls' ],
     port: 9876,
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
     browsers: ['Chrome'],
-    singleRun: false
+    singleRun: !argv.watch,
```

Comment:
I'm not sure i follow this line. `singleRun` is the default right? The `watch` arg is overwritting the default, why do we need to have it here as well?

Reply:
Yeah, maybe I should delete autowatch in favor of singleRun by arguments. What do you think?

Reply:
What's the worry here? Trying to understand why we need to change singleRun at all

Reply:
isn't this what the `--watch` flag will do?

Reply:
So why do they define two different properties for the same purpose? 😕

Reply:
Yeah it's weird. I saw bugs where you can't have both `singleRun` and `autoWatch` set to true. 🤷‍♂️ 

I think everything else looks good from my point of view. Let's make this change and get some code coverage action!

Reply:
- `singleRun`: Continuous Integration mode.
If true, Karma will start and capture all configured browsers, run tests and then exit with an exit code of 0 or 1 depending on whether all tests passed or any tests failed.
- `autoWatch`: Enable or disable watching files and executing the tests whenever one of these files changes.

Reply:
Which flag do I delete?

Reply:
I was thinking that we don't need the `!argv.watch`. Maybe i'm misunderstanding something. Let's chat today or Tuesday (long-weekend here)?

Reply:
I'm abroad until next Thursday, I'll make tests with and without the flag and let you know the best result and then we can merge or chat if we need to sync😁🙌🙌 sounds good to you?